WooCommerce/Classes/Blaze/BlazeCampaignCreationCoordinator.swift

```diff
@@ -57,25 +59,60 @@ final class BlazeCampaignCreationCoordinator: Coordinator {
         self.navigationController = navigationController
         self.didSelectCreateCampaign = didSelectCreateCampaign
         self.onCampaignCreated = onCampaignCreated
+        self.analytics = analytics
 
         configureResultsController()
     }
 
-    func start() {
-        switch blazeCreationEntryDestination {
-        case .productSelector:
-            navigateToBlazeProductSelector(source: source)
-        case .campaignForm(let productID):
-            navigateToNativeCampaignCreation(source: source, productID: productID)
-        case .webViewForm(let productID):
-            navigateToWebCampaignCreation(source: source, productID: productID)
-        case .noProductAvailable:
-            presentNoProductAlert()
+    func start(shouldShowIntro: Bool = false) {
```

Comment:
Can we move the `shouldShowIntro` parameter to `BlazeCampaignCreationCoordinator`'s `init` method instead? This way, we can keep the `Coordinator` conformance.

Reply:
Good point! Updated in 11d904a2fe18f519a303b089db593adda0c240ed.